File: java/Factorial.java
    import java.util.*;
    
    class Factorial{  
     public static void main(String args[]){  
      Scanner sc = new Scanner(System.in);
      int i,fact=1;  
      int number=sc.nextInt();//It is the number to calculate factorial    
      for(i=1;i<=number;i++){    
          fact=fact*i;    
      }    
      System.out.println("Factorial of "+number+" is: "+fact);    
     }  
    }  

File: php/factorial.php
<?php

/**
* Calculate a factorial given a number.
* For different results, change the function call.
*/
function factorial($number) { 

    if ($number < 2) { 
        return 1; 
    } else { 
        return ($number * factorial($number-1)); 
    } 
}

$res = factorial(4);
echo $res;
